### PR TITLE
Fix: Add more detailed documentation for the Api

### PR DESCRIPTION
**Pull request by nassersaazi**
- Documentation can now be accessed on its own route i.e. `api/docs`
- Api development port has been changed from 80 to 8080(since 80 is the default port for Apache/Nginx)
- README has been updated accordingly

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -15,13 +19,13 @@ const db = require("./app/models");
...
-    console.log("All models were synchronised successfully.");
-  })
-  .catch((err) => {
-    console.log("Failed to sync db: " + err.message);
-  });
+//db.sequelize.sync()
+//  .then(() => {
+//    console.log("All models were synchronised successfully.");
+//  })
+//  .catch((err) => {
```
Comment (adamjonas) [inline api/server.js:26]: Should we just delete rather than comment out?

File: README.md [hunk 2]
```diff
@@ -16,13 +16,5 @@
...
-#### GET requests
-
--   `http://localhost:8080/api/transcripts`
--   `http://localhost:8080/api/users`
-
-### TODOS:
-- // Add swagger api documentation
-- // Set up the application on AWS
-- // Supply more meaningful data in the init.sql file
+- Access the endpoints documentation here: `http://localhost:8080/api/docs`
```
Comment (adamjonas) [inline README.md:19]: Tried to access at 8080/api/docs and nothing loaded.

Screen Shot 2023-03-10 at 6 59 33 PM

File: docker-compose.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ services:
     volumes:
       - ./api:/src
     ports:
-      - "80:8080"
+      - "8080:8080"
```
Comment (adamjonas) [inline docker-compose.yml:25]: is this being accounted for on the front end?

File: api/server.js (hunk 1 above)
Comment (nassersaazi) [reply api/server.js:26]: I left that out as it was useful when syncing in cases schema design changes. but will delete and refer to earlier commits instead

File: docker-compose.yml (hunk 3 above)
Comment (nassersaazi) [reply docker-compose.yml:25]: @Emmanuel-Develops what do you think? i thought it might be better to switch away from the default Apache/Nginx port

Comment (Emmanuel-Develops) [reply docker-compose.yml:25]: Either one works for me. I'll simply update on my ends

File: README.md (hunk 2 above)
Comment (Emmanuel-Develops) [reply README.md:19]: I can see the docs
Screenshot 2023-03-13 at 8 38 24 PM

**Review by Emmanuel-Develops — APPROVED** [review]

**Pull request merged**